docopt.cpp

```diff
@@ -17,11 +17,26 @@
 #include <unordered_map>
 #include <map>
 #include <string>
-#include <regex>
 #include <iostream>
 #include <cassert>
 #include <cstddef>
 
+// Workaround GCC 4.8 not having std::regex
+#if (__GNUC__ == 4) && (__GNUC_MINOR__ == 8)
```

Comment:
Perhaps we just have "#if DOCTOPT_USE_BOOST_REGEX" without the version check? If people really want to use GCC 4.8 then it would be on them to fix up the flag rather than us trying to guard it.